src/protocols/screencopy.rs

```diff
@@ -187,24 +200,23 @@ where
             buffer_size.w as u32 * 4,
         );
 
-        // if manager.version() >= 3 {
-        //     // Send desired DMA buffer parameters.
-        //     frame.linux_dmabuf(
-        //         Fourcc::Argb8888 as u32,
-        //         buffer_size.w as u32,
-        //         buffer_size.h as u32,
-        //     );
-        //
-        //     // Notify client that all supported buffers were enumerated.
-        //     frame.buffer_done();
-        // }
+        // Send desired DMA buffer parameters.
+        frame.linux_dmabuf(
+            Fourcc::Argb8888 as u32,
```

Comment:
I think we want at least Xrgb too here. Might be worth checking what formats clients typically query.

Reply:
`wf-recorder` accepts a number of [formats](https://github.com/ammen99/wf-recorder/blob/9978c3570436eb5358e4578421d83a7f3ffd2683/src/frame-writer.cpp#L149). I will add XRGB8888 for now.

Reply:
The documentation of the protocol seems to suggest that only one format can be supported by the compositor:
> Provides information about linux-dmabuf buffer parameters that need to be used for this frame. This event is sent _once_ after the frame is created if linux-dmabuf buffers are supported.

Reply:
Hm, okay then. I'm just thinking that maybe Xrgb is a better choice, but idk. Might be worth checking what sway sends in that case